backend/open_webui/routers/auths.py

```
@@ -397,111 +401,257 @@ async def login_auth0(request: Request):
 
 
 @router.get("/oauth/auth0/callback")
-async def auth0_callback(request: Request, response: Response, code: str = None, state: str = None):
+async def auth0_callback(
```

Ruff (0.8.2)

`auth0_callback` is too complex (32 > 10)

(C901)